signac/common/validate.py

```diff
@@ -5,53 +5,16 @@
 
 import logging
 
-from .configobj.validate import Validator, VdtValueError
+from .configobj.validate import Validator
 
 logger = logging.getLogger(__name__)
 
 
-def mongodb_uri(value, *args, **kwargs):
-    """Return a MongoDB URI."""
-    if isinstance(value, list):
-        value = ",".join(value)
-    if not value.startswith("mongodb://"):
-        value = "mongodb://" + value
-    try:
-        import pymongo
-    except ImportError:
-        logger.debug("Install pymongo to validate database configurations!")
-    else:
-        try:
-            pymongo.uri_parser.parse_uri(value)
-        except pymongo.errors.InvalidURI:
-            raise VdtValueError(value)
-    return value
-
-
-def password(value, *args, **kwargs):  # noqa: D103
-    return value
-
-
 def get_validator():  # noqa: D103
-    return Validator({"mongodb_uri": mongodb_uri, "password": password})
+    return Validator()
 
 
+# TODO: Rename to something internal and uppercase e.g. _CFG.
```

Comment:
Future work: reminder to grep for `TODO`s like this. Several of them would be easy to address.